data-processing-lib/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [project]
 name = "data_prep_lab"
-version = "0.1.5"
+version = "0.1.6"
```

Comment:
You do not really need to do this. It should update automatically based on make.versions